tests/integration/it_utils.py

```diff
@@ -124,6 +158,36 @@ async def sign_and_reliable_submission_async(
     return response
 
 
+def accept_ledger(
+    use_json_client: bool = True, timeout: float = LEDGER_ACCEPT_TIME
+) -> None:
+    """
+    Allows integration tests for sync clients to send a `ledger_accept` request
+    after a set period of time.
+
+    Arguments:
+        use_json_client: boolean for choosing json or websocket client.
+        timeout: float for how long to wait before accepting ledger.
```

Comment:
You should add the units here :) 
```suggestion
        timeout: float for how many seconds to wait before accepting ledger.
```

Reply:
(Same down below)